src/components/ContactList/ContactList.tsx

```diff
@@ -8,12 +8,32 @@ import ContactModalBody from "../ContactModal/Body";
 interface Props {
     contacts: Contact[],
     isLoading: boolean,
+    onScroll: () => void,
 }
 
-const ContactList = forwardRef((props: Props, ref: ForwardedRef<any>) => {
-    const { contacts, isLoading } = props;
+const ContactList = (props: Props) => {
+    const { contacts, isLoading, onScroll } = props;
```

Comment:
incearca sa fi mai specific la denumiri, onLoadMore in loc de scroll. onScroll se intelege ca se declanseaza in fiecare moment cand se face scroll. Insa treaba asta se declanseaza doar cand a ajuns la bottom

Reply:
Reminder pe aceasta modificare